packages/base/card-api.gts

```diff
@@ -145,6 +147,8 @@ export interface CardContext {
     };
   }>;
   prerenderedCardSearchComponent: any;
+  getCard: getCard<T>;
+  getCards: getCards;
```

Comment:
There is additionally a getCards inside of actions. Whats the distinction between getCards here and getCards actions? 

Thanks for adding tho, I always thought not having `getCard` was weird

Reply:
the action for getCards will be going away in the next PR